src/scss/_standard.scss

```diff
@@ -1,27 +1,161 @@
+////
+/// @group o-card base styles
+/// @link http://registry.origami.ft.com/components/o-card
+////
+
+/// Base card styles
 @mixin oCard {
 	@include oColorsFor(o-card box, background);
-	height: 100%;
-	position: relative;
+	width: 100%;
+	padding: $o-card-spacing-unit;
+	padding-bottom: 0;
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+}
+
+/// Base tag styles
+@mixin oCardTag {
+	@include oColorsFor(o-card-tag, text);
+	@include oTypographySansBold(s);
+	display: block;
+	margin: 0 0 $o-card-spacing-unit;
+	// If the tag is used as a link
+	text-decoration: none;
 
-	display: inline-block;
-	// Force this element to have layout in ie6/7 so we can use inline-block
-	*display: inline;
-  zoom: 1;
+	#{$o-hoverable-if-hover-enabled} &:hover {
+		@include oColorsFor(o-card-tag-hover, text);
+	}
 }
 
-@mixin oCardContainer {
-	float: left;
-	padding: $o-card-spacing-unit/2;
+/// Base timestamp styles
+@mixin oCardTimestamp {
+	@include oColorsFor(o-card-timestamp, text);
+	@include oTypographySans(xs);
+	display: block;
+	margin: $o-card-spacing-unit 0;
+	text-transform: uppercase;
+}
+
+/// Base heading styles (includes link styles within the element)
+@mixin oCardHeading {
+	@include oTypographyHeading2;
+	@include oColorsFor(link-title, text);
+	margin-bottom: $o-card-spacing-unit;
+
+	&--small {
+		@include oTypographyHeading3;
+	}
+
+	// If there is a link in the heading
+	a {
+		color: inherit;
+		text-decoration: none;
+	}
+
+	#{$o-hoverable-if-hover-enabled} a:hover {
+		border-bottom: 1px dotted;
```

Comment:
yup, good idea

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*